Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -108,51 +108,62 @@ def signoff!(path, pr: nil, dry_run: false)
     end
   end
 
-  def determine_bump_subject(old_contents, new_contents, formula_path, reason: nil)
-    formula_path = Pathname(formula_path)
-    formula_name = formula_path.basename.to_s.chomp(".rb")
+  def get_package(tap, subject_name, subject_path, content)
+    if subject_path.dirname == tap.cask_dir
+      begin
+        Cask::CaskLoader.load(content.dup, config: Cask::Config.defaults)
+      rescue Cask::CaskUnavailableError => e
+        ohai e
+        nil
+      end
+    end
```

Comment:
Does anyone has any ideas why this will always return `nil` if it gets a Cask content? It seems to be using the nullloader instead of `FromContentLoader` but I don't know how to force one.

Reply:
Not sure, sorry!